src/main/java/net/minestom/server/terminal/MinestomTerminal.java

```diff
@@ -58,7 +68,39 @@ public static void stop() {
             } catch (IOException e) {
                 e.printStackTrace();
             }
+            reader = null;
         }
     }
 
+    @Nullable
+    public static LineReader getReader() {
```

Comment:
What is this getter for?

Reply:
@TheMode 
For `MinestomConsoleWriter`.
I explained in the above review.

Reply:
Switched to `package-private`